docs/references/filters.md

```diff
@@ -79,6 +79,24 @@ class FruitFilter:
         return queryset.exclude(name='banana')
 ```
 
+## Custom filter logic
+
+For overriding the default filter logic you can provide the filter method.
+Other filter methods are not executed anymore (default as well as custom).
```

Comment:
I would be more clear here of what exactly does it mean to override the `filter` method. Not everyone should be doing it IMO.

Don't know if the suggestion here is enough (feel free to suggest something different)

```suggestion
Note that no default filtering will be done anymore, which means your custom
method is responsible for handling all the all filter related operations.
```